catalog/static/config.js

```diff
@@ -1,10 +1,12 @@
-/* eslint-disable */
+/* config.js */
+
+/* eslint-disable no-underscore-dangle */
 if (window.location.hostname === 'quiltdata.com') {
   window.__CONFIG = {
     api: 'https://pkg.quiltdata.com',
     stripeKey: 'pk_live_aV44tCGHpBZr5FfFCUqbXqid',
     userApi: 'https://app.quiltdata.com/accounts/api-root',
-    signOutUrl: 'https://app.quiltdata.com/accounts/logout?next=%2F'
+    signOutUrl: 'https://app.quiltdata.com/accounts/logout?next=%2F',
```

Comment:
Technically, this is wrong :) Probably breaks IE8 (not that we really care)

Reply:
oh yeah `eslint` loves it though; there *might* be a babel transformer for it. we set IE support at > 10 so should be ok.

Reply:
actually i'll just fix this